.github/workflows/publish.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           fi
           
           # Always build sitedoc because all other changes affect it
-          ./sitedoc.sh ${{ github.ref_name }}
+          ./sitedoc.sh ${{ github.event.pull_request.base.ref }}
```

Comment:
You probably don't want to mess with that. So many things can break when we change this (revsys, previews, tests, etc...).

Reply:
@alandefreitas `${{ github.ref_name }}` will not work on pull requests. As you can see in below image it hold pull reference instead of branch reference. After I change it to `${{ github.event.pull_request.base.ref }}` now we can get target branch `develop` but `Edit this Page` link is still referring to `HEAD` branch IDK why. It's working fine on my local.
<img width="789" alt="image" src="https://github.com/cppalliance/site-docs/assets/40260519/85b51de9-05d4-4a3b-ab64-fa38cc3bbbfb">

Reply:
After this workflow change you can see the correct branch name.
<img width="789" alt="image" src="https://github.com/cppalliance/site-docs/assets/40260519/bc6995d4-f56d-4884-84d9-f0390533467d">

Reply:
And obviously we need to test it that it will works as it in all cases.

Reply:
@alandefreitas I have made some changes for redirecting to develop branch from Edit this Page link. See if it looks good to you.

Reply:
The link in the preview is now https://github.com/cppalliance/site-docs/edit/develop/user-guide/modules/ROOT/pages/index.adoc

Does it look different locally?

Reply:
No! it redirects to develop locally too.